<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>41</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>257</width>
    <height>41</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pbtnShotAndMin">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>9</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>最小化截图</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnMin">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>9</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>最小化</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtnShot">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>截   图</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pbtnShot</tabstop>
  <tabstop>pbtnShotAndMin</tabstop>
  <tabstop>pbtnMin</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
